app/views/result_filters/qualification/new.html.erb

```diff
@@ -0,0 +1,120 @@
+<%= content_for :page_title, "Filter by qualification" %>
+<% content_for :before_content do %>
+  <%= govuk_back_link_to(results_path(request.query_parameters), "Back to search results") %>
+<% end %>
+
+<% if flash[:error] %>
+  <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary" data-ga-event-form="error" data-qa="error">
+    <h2 class="govuk-error-summary__title" id="error-summary-title">
+      There is a problem
```

Comment:
```suggestion
You’ll need to correct some information. 
```

![image](https://user-images.githubusercontent.com/470137/72990647-7140ca80-3de8-11ea-9357-f324285cc021.png)
If the behaviour is correct than language used is questionable and seems aggressive...

`You must` vs `Please make` at least one selection.

`There is a problem` tell user there is a problem and nothing more without `full stop`.
` You’ll need to correct some information.` Tell user to do something correctly...

Are we sure:
- [ ] The wording/text
- [ ] Full width red border/styling

Which ever wording we use, can we have a test for the exacting wording as the error/validation display mechanism going to be reused...

Reply:
Updated in 1758477.

<img width="881" alt="Screenshot 2020-01-23 at 14 45 10" src="https://user-images.githubusercontent.com/5216/72994423-fd55f080-3dee-11ea-81cf-b0a7567c205d.png">

Reply:
I think one of the other filters will need updating too but I'll do that in a separate PR.